add script-dump document (#143)
valkey-io/valkey-doc: commands/script-show.md

Returns the original source code of a script in the script cache.

This command accepts a SHA1 digest and returns the original script's source code if the script is present in the script cache.

> This command is intended primarily for debugging, to introspect into the contents of a script when the user does not have access to script anymore.

Reviewer: ```suggestion
It is intended primary for debugging, allowing users to introspect the contents of a script when they no longer have direct access to it.
```

> One example is when an admin only has access to a script SHA1 from monitor or slowlog, and is unable to determine the contents of the script.

Reviewer: ```suggestion
For example, an admin may only have access to a script's SHA1 from the monitor or slowlog and needs to determine the script's contents.
```
